pkg/ibmvpc/examples/acl_testing5_oldsubnetsBased_withPGW.txt

```diff
@@ -1,11 +1,13 @@
 combined connections between subnets:
-sub1-1-ky => Public Internet [8.8.8.8/32] : protocol: UDP dst-ports: 53
+sub1-1-ky => Public Internet 8.8.8.8/32 : protocol: UDP dst-ports: 53
 sub1-1-ky => sub1-2-ky : protocol: TCP
 sub1-1-ky => sub1-3-ky : protocol: TCP
 sub1-2-ky => sub1-1-ky : protocol: TCP
 sub1-2-ky => sub1-3-ky : protocol: TCP
 sub1-3-ky => sub1-1-ky : protocol: TCP
 sub1-3-ky => sub1-2-ky : protocol: TCP
-sub2-1-ky => Public Internet [8.8.8.8/32] : protocol: UDP dst-ports: 53
+sub2-1-ky => Public Internet 8.8.8.8/32 : protocol: UDP dst-ports: 53
 sub2-1-ky => sub2-2-ky : All Connections
 sub2-2-ky => sub2-1-ky : All Connections
+
+connections are stateful unless marked with *
```

Comment:
we do not yet have any implemented computation of whether subnets connectivity is stateful.
We should remove that comment , and have another issue to handle stateful computation for this as well,

Reply:
Opened https://github.com/np-guard/vpc-network-config-analyzer/issues/138 (next in my todos) as well as ugly WA code that will be removed once the issue is solved